### PR TITLE
Fixes #9723 - missing owner_type in host api docs

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

**Comment by tstrachota** [comment]
@domcleal is it worth cherry-picking into 1.8?
(+1 from my side to update params in hammer)

**Comment by domcleal** [comment]
@tstrachota yep, I almost certainly will

File: app/controllers/api/v2/hosts_controller.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ def show
           param :model_id, :number
           param :hostgroup_id, :number
           param :owner_id, :number
+          param :owner_type, Host::Base::OWNER_TYPES, :desc => N_("Host's owner type")
```
Comment (ares) [inline app/controllers/api/v2/hosts_controller.rb:59]: maybe valid values in description would be good

Comment (tstrachota) [inline app/controllers/api/v2/hosts_controller.rb:59]: apipie displays the valid options in value section "Value: Must be one of: ..."

Comment (ares) [inline app/controllers/api/v2/hosts_controller.rb:59]: OK, I didn't realize this. Thanks for clarification.

**Comment by ares** [comment]
Merged as 55be35e9699765bb5fb137db518b06411be20939, thanks @tstrachota!

**Pull request closed without merging**